Automattic/wc-calypso-bridge
# Replace tax task to remove Avalara

### Changes proposed in this Pull Request:



Closes https://github.com/Automattic/wc-calypso-bridge/issues/972. This PR:

1. Imports Tax task fill and replace the original fill with Avalara removed
2. Added shared variables and mixin stylesheets
3. Added some JS utils



### How to test the changes in this Pull Request:

**Test Avalara not suggested**
1. Make sure your store is located in Avalara Tax supported country, such as `US`
1. Go to WooCommerce > Home > Tax task
1. Observe that Avalara is not suggested

**Test WooCommerce Tax**
1. Click on "Yes please" and observe that you're redirected to Homescreen with Tax task completed
2. Go to WooCommerce > Settings > Tax
3. Observe that you have `Enable automated taxes` selected in the `Automated taxes` dropdown
1. Go to WooCommerce > Home > Tax task
2. Click on `No thanks, I'll set up manually` and observe that you're redirected to tax settings screen
1. Go to WooCommerce > Home > Tax task
2. Click on `I don't charge sales tax`
2. Observe that you're redirected to Homescreen with Tax task completed


**Test manual configuration**
1. Go to WooCommerce > Settings > General
5. Remove all address fields, set store country to "Malaysia - Johor" and save
6. Go to WooCommerce > Home > Tax task
7. Fill in all store address fields in `Set store location` and click on continue
8. Click on `Configure` and observe that you're redirected to tax settings
9. Go to WooCommerce > Settings > General
10. Observe that the store address values are saved from the values inserted in the previous form

### Screenshots

#### WooCommerce Tax:
<img width="826" alt="image" src="https://user-images.githubusercontent.com/3747241/220317274-73a1ba88-4e6d-4481-8700-552edbc18f68.png">

#### Manual configuration:
<img width="749" alt="image" src="https://user-images.githubusercontent.com/3747241/220318802-0861ca6f-948f-43ba-97c2-506aded0b40e.png">
